### PR TITLE
[ELY-2581] Update Elytron's SECURITY.md file

### PR DESCRIPTION
**Pull request by ivassile**
Issue: https://issues.redhat.com/browse/ELY-2581

### CODE REVIEW

**Comment by ivassile** [comment]
@fjuma What is the email address to report CVEs?

File: SECURITY.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+
+### Email the mailing list</h2>
+
+The list at <a href="mailto:security.wildfly.org">security.wildfly.org</a> is the preferred mechanism for outside users to report security issues. A member of the WildFly Elytron team will open the required issues.
+    
+### Other considerations</h2>
+
+If you would like to work with us on a fix for the security vulnerability, please include your GitHub username in the above email, and we will provide you access to a temporary private fork where we can collaborate on a fix without it being disclosed publicly, **including in your own publicly visible git repository**.
+
+Do not open a public issue, send a pull request, or disclose any information about the suspected vulnerability publicly, **including in your own publicly visible git repository**. If you discover any publicly disclosed security vulnerabilities, please notify us immediately through <a href="mailto:security.wildfly.org">security.wildfy.org
```
Comment (fjuma) [inline SECURITY.md]: The email address to use is security@wildfly.org.

File: SECURITY.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+
+## Reporting of Security Issues
+
+When reporting a security vulnerability it is important to not accidentally broadcast to the world that the issue exists, as this makes it easier for people to exploit it. The software industry uses the term <a href="https://www.redhat.com/en/blog/security-embargoes-red-hat">embargo</a> to describe the time a security issue is known internally until it is public knowledge.
+
+Our preferred way of reporting security issues in WildFly Elytron and its related projects is listed below.
+
+### Email the mailing list</h2>
+
+The list at <a href="mailto:security.wildfly.org">security.wildfly.org</a> is the preferred mechanism for outside users to report security issues. A member of the WildFly Elytron team will open the required issues.
```
Comment (fjuma) [inline SECURITY.md]: Same here.

**Comment by fjuma** [comment]
> @fjuma What is the email address to report CVEs?

Sorry, I somehow missed this question! The email address is security@wildfly.org.

**Comment by ivassile** [comment]
Corrected the email and rebased on 2.x

**Review by fjuma — APPROVED** [review]
Thanks @ivassile!

**Comment by fjuma** [comment]
@Skyllarr Please review.

**Review by skyllarr — APPROVED** [review]

**Comment by skyllarr** [comment]
@ivassile Thanks!

**Pull request merged**